### PR TITLE
UPGRADE to Babel 7

### PR DESCRIPTION
**Pull request by sarovin**

### CODE REVIEW

**Comment by sarovin** [comment]
@n6g7 I don't understand the circleci error, can you help me with this?
This PR is important for work with nextjs@6

File: .babelrc [hunk 1]
```diff
@@ -1,16 +1,9 @@
 {
-  "presets": [
-    "es2015",
-    "stage-3"
-  ],
-  "plugins": [
-    "transform-runtime"
-  ],
+  "presets": ["@babel/es2015", "@babel/stage-3"],
```
Comment (n6g7) [inline .babelrc]: Could you also replace `preset-es2015` with `preset-env` please?
I've been meaning to do it for a while and that's a good opportunity to do so? :)

**Comment by n6g7** [comment]
Looks like you need to install `babel-core@^7.0.0-0` as well: https://github.com/facebook/jest/issues/5525#issuecomment-365095681.
That's weird ... 🤔

File: .babelrc (hunk 1 above)
Comment (sarovin) [reply .babelrc]: of course, good idea

**Comment by sarovin** [comment]
@n6g7 Please, not merge...I must test this code

File: .babelrc [hunk 2]
```diff
@@ -1,16 +1,9 @@
 {
-  "presets": [
-    "es2015",
-    "stage-3"
-  ],
-  "plugins": [
-    "transform-runtime"
-  ],
+  "presets": ["@babel/preset-env"],
```
Comment (n6g7) [inline .babelrc]: I think you'll need to keep `preset-stage-3`. ;)

**Comment by n6g7** [comment]
@sarovin I rebased your commits and merged https://github.com/n6g7/redux-saga-firebase/pull/113 instead with your changes. I'll close this PR.

Thanks a lot for these changes, as always! ;)

**Pull request closed without merging**